packages/components/resource-pages/ResourcePopOver.tsx

```diff
@@ -1,24 +1,40 @@
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
-import { Button, Popover, PopoverPosition } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
 import {
+  Button,
+  Divider,
+  Popover,
+  PopoverPosition,
+} from '@patternfly/react-core';
+import {
+  NooBaaBucketClassModel,
   NooBaaNamespaceStoreModel,
   NooBaaObjectBucketClaimModel,
 } from '../../models';
 import { referenceForModel } from '../../utils';
 import ResourceLink from '../../utils/generics/resource-link';
 import './resources.scss';
 
+const OBC_LIST_PATH =
+  '/k8s/all-namespaces/objectbucket.io~v1alpha1~ObjectBucketClaim';
+const BUCKET_CLASS_LIST_PATH =
+  '/mcgms/cluster/resource/noobaa.io~v1alpha1~BucketClass';
+const DATA_RESOURCE_LIST_PATH =
+  '/mcgms/cluster/resource/noobaa.io~v1alpha1~NamespaceStore';
```

Comment:
if these are used in multiple files.... move them to constants folder and import from there...